code/modules/mob/living/carbon/carbon.dm

```diff
@@ -25,21 +25,6 @@
 		qdel(dna)
 	return ..()
 
-/mob/living/carbon/Move(NewLoc, direct)
-	. = ..()
-	if(.)
-		if(src.nutrition && src.stat != 2)
-			src.nutrition -= HUNGER_FACTOR/10
-			if(src.m_intent == "run")
-				src.nutrition -= HUNGER_FACTOR/10
-		if((src.disabilities & FAT) && src.m_intent == "run" && src.bodytemperature <= 360)
-			src.bodytemperature += 2
-
-/mob/living/carbon/movement_delay()
-	. = ..()
-	if(legcuffed)
-		. += legcuffed.slowdown
-
 /mob/living/carbon/relaymove(mob/user, direction)

```

Comment:
relaymove() is not really movement. It's something trying to move inside you. Are you sure it fits in `carbon_movement.dm`?